docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+
+services:
+  webapp-prod:
+    container_name: production
+    build:
+      dockerfile:  Dockerfile
+      context: .
+      target: production
+    env_file:
+      - .env
+    ports:
+      - 80:80
+  webapp-dev:
+    container_name: development
+    build:
+      dockerfile: Dockerfile
+      context: .
+      target: development
+    volumes:
+      - type: bind
+        source: ./todo_app
+        target: /opt/todo_app
+    env_file:
+      - .env.dev
```

Comment:
Instead of having a separate env file, if there are just a few values you want to change, you can set those in the compose file. E.g.
```yaml
    environment:
      - FLASK_ENV=development
    env_file:
      - .env
```

Values set in the compose file will take precedence over those in the env file.